perfkitbenchmarker/providers/gcp/gcp_spanner.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for GCP's spanner instances.
+
+Instances can be created and deleted.
+"""
+
+import logging
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+from perfkitbenchmarker.providers.gcp import util
+
+FLAGS = flags.FLAGS
+
+
+DEFAULT_CLOUD_SPANNER_END_POINT = 'https://spanner.googleapis.com/'
+DEFAULT_CLOUD_SPANNER_CONFIG = 'regional-us-central1'
```

Comment:
This is the default value to the class constructor. Benchmarks can override it by passing a different value to the constructor.